website/api/helpers/strings/to-html.js

```diff
@@ -112,6 +112,11 @@ module.exports = {
       return `<blockquote><img src="/images/icon-info-16x16@2x.png" alt="An icon indicating that this section has important information"><div class="d-block">`+quote+`</div></blockquote>\n`;
     };
 
+    // Creating a custom table renderer to add Bootstrap's responsive table styles to markdown tables.
+    customRenderer.table = function(header, body) {
+      return `<div class="table-responsive-xl"><table class="table">\n<thead>\n${header}\n</thead>\n<tbody>${body}\n</tbody>\n</table>\n</div>`;
+    };
+
```

Comment:
tip: consider naming these `headerHtml` and `bodyHtml` so that future paranoid Mikes don't quiz you about whether it's HTML or plaintext and whether HTML escaping is appropriate